src/components/attendee/errors/Forbidden.tsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+import { BlankPage } from "./BlankPage";
+
+export const Forbidden: React.FC = () => (
+  <BlankPage>
+    <h1 data-bem="Forbidden__title" data-side="att">
+      403 Not authorised
```

Comment:
We might want to have a shared config file with these error codes and description to use in both admin and attendee side. It's currently used in 2 places, so keeping it all in a single config would be a good thing

Reply:
They aren't codes but a display text that will most likely be replaced after the new login flow is implemented. Maybe even different for attendee and admin sides.